djstripe/models.py

```diff
@@ -74,34 +56,44 @@ def __str__(self):
         return "<{message}, pk={pk}, Event={event}>".format(message=self.message, pk=self.pk, event=self.event)
 
 
-@python_2_unicode_compatible
-class Event(StripeObject):
+class Event(StripeEvent):
+    objects = models.Manager()
+    customer = models.ForeignKey("Customer", null=True,
+                                 help_text="In the event that there is a related customer, this will point to that "
+                                           "Customer record")
+    valid = models.NullBooleanField(null=True,
+                                    help_text="Tri-state bool. Null == validity not yet confirmed. Otherwise, this "
+                                              "field indicates that this event was checked via stripe api and found "
+                                              "to be either authentic (valid=True) or in-authentic (possibly "
+                                              "malicious)")
 
-    kind = models.CharField(max_length=250)
-    livemode = models.BooleanField(default=False)
-    customer = models.ForeignKey("Customer", null=True)
-    webhook_message = JSONField()
-    validated_message = JSONField(null=True)
-    valid = models.NullBooleanField(null=True)
-    processed = models.BooleanField(default=False)
+    processed = models.BooleanField(default=False, help_text="If validity is performed, webhook event processor(s) "
+                                                             "may run to take further action on the event. Once these "
+                                                             "have run, this is set to True.")
 
     @property
     def message(self):
-        return self.validated_message
-
-    def __str__(self):
-        return "<{kind}, stripe_id={stripe_id}>".format(kind=self.kind, stripe_id=self.stripe_id)
+        if self.valid:
+            return self.webhook_message
```

Comment:
How do I feel about it? I actually find it a little harder to read usually. But in this case I'll just switch it to ternary